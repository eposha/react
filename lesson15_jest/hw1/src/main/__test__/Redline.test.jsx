import React from "react";
import { shallow } from "enzyme";
import Redline from "../Redline";

describe("<Redline/ >", () => {
  it("should display Redline", () => {
    const wrappedComponent = shallow(<Redline />);
    expect(wrappedComponent).toMatchSnapshot();
  });
});
