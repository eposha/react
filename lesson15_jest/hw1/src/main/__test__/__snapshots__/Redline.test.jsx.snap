// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Redline/ > should display Redline 1`] = `
<div
  className="red-line"
  style={
    Object {
      "top": "1384px",
    }
  }
/>
`;
